build.sbt

```diff
@@ -0,0 +1,35 @@
+
+name := "neo4j-scala-client"
+
+organization := "me.jeffmay"
+
+scalaVersion := "2.11.7"
+
+val playVersion = "2.4.6"
+val playJsonOpsVersion = "1.1.5"
+val scalatestVersion = "3.0.0-M10"
+
+resolvers ++= Seq(
+  "Artima Maven Repository" at "http://repo.artima.com/releases",
+  "jeffmay" at "http://dl.bintray.com/jeffmay/maven"
+)
+
+libraryDependencies ++= Seq(
+  "com.typesafe.play" %% "play-json" % playVersion,
+  "com.typesafe.play" %% "play-ws" % playVersion,
+  "me.jeffmay" %% "play-json-ops" % playJsonOpsVersion,
+  "me.jeffmay" %% "play-json-tests" % playJsonOpsVersion % "test",
+  "org.scalactic" %% "scalactic" % scalatestVersion,
+  "org.scalatest" %% "scalatest" % scalatestVersion % "test"
+)
+
+dependencyOverrides := Set(
+  "com.typesafe.play" %% "play-json" % playVersion,
+  "com.google.guava" % "guava" % "18.0",
+  "org.scalactic" %% "scalactic" % scalatestVersion
+)
+
+testOptions in Test += Tests.Cleanup({
+  loader: java.lang.ClassLoader =>
+    loader.loadClass("me.jeffmay.util.TestCleanupInParallel").newInstance
```

Comment:
This cleans up any objects that won't get garbage collected from sbt (hopefully). Closes the global TestWS client, etc.